ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -26,18 +25,21 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   } = props;
 
   const suggestedStepMs = useSuggestedStepMs(contentDimensions?.width);
+  const queryResults = useTimeSeriesQueries(queries, { suggestedStepMs });
 
+  if (contentDimensions === undefined) {
+    return null;
+  }
+
+  // TODO: Do we need this container component any more now that we can run multiple queries here?
```

Comment:
We shouldn't need it now! This extra component has been bothering me for awhile so excited to get rid of it 🎉